app/src/main/AndroidManifest.xml

```diff
@@ -14,7 +13,15 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         android:usesCleartextTraffic="true">
-        <activity android:name=".presentation.MainActivity">
+        <activity android:name=".presentation.MarvelLandingScreen"
+            android:theme="@style/LandingScreen">
+            <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
+            <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+        <activity android:name=".presentation.MainActivity"
+            android:theme="@style/Theme.AppCompat.NoActionBar">
             <intent-filter>
```

Comment:
remove this <intent-filter>. we only have one LAUNCHER screen and, in your case, is not MainActivity anymore, now it is LandingScreen

Reply:
done